.ado/jobs/e2e-test.yml

```diff
@@ -56,10 +56,25 @@ jobs:
         condition: false # Must be manually enabled, since it causes a 5x perf reduction that causes test instability
 
       - task: CmdLine@2
-        displayName: run-windows
+        displayName: run-windows - PR
         inputs:
-          script: yarn windows --no-launch --no-packager --arch $(BuildPlatform) --release --logging --buildLogDirectory $(BuildLogDirectory) --msbuildprops BaseIntDir=$(BaseIntDir)
+          script: yarn windows --arch $(BuildPlatform) --logging --buildLogDirectory $(BuildLogDirectory) --msbuildprops BaseIntDir=$(BaseIntDir) AppxPackageSigningEnabled=false
           workingDirectory: packages/e2e-test-app
+        condition: eq('${{ parameters.BuildEnvironment }}', 'PullRequest')
+
+      - task: DownloadSecureFile@1
```

Comment:
I saw you added them to ADO, I tried to read docs, but it would be great if we could use the KeyVault for this. 
The reasons for that would be:
* The internal builds on the windows account would now need a copy we have to manually sync between the two...
* There is no expiration management in ADO, there is in KeyVault
* There is no history in ADO, KeyVault has proper versioning of secrets. in ADO there is no way to extract the file as a backup. Say we try to rotate the secret before it expires, that somehow goes wrong. Since we can't download the file from ADO GUI we are now in a broken state with no option of 'rolling back'...